pywemo/ouimeaux_device/bridge.py

```diff
@@ -105,7 +105,7 @@ def _update_state(self, status):
             if not value:
                 value = None
             elif ':' in value:
-                value = tuple(int(v) for v in value.split(':'))
+                value = tuple(int(float(v)) for v in value.split(':'))
```

Comment:
Probably best to do `int(round(float(v)))`  which will round rather than truncate - and work on Python 2 & 3.